assets/section-footer.css

```diff
@@ -29,12 +32,14 @@
   display: block;
 }
 
-@media screen and (min-width: 750px) {
+@media screen and (max-width: 749px) {
   .footer__content-top {
```

Comment:
Maybe we could move this to the media query above as it's sharing the same breakpoint.